packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/server/dashboard/dashboard.controller.js

```diff
@@ -0,0 +1,133 @@
+import {
+  COMMIT_IMPRESSION_TRACKING_DATA,
+  RECOMMIT_IMPRESSION_TRACKING_DATA,
+  HIDE_MRTG_FOR_SERVER_RANGES,
+} from './dashboard.constants';
+import { NEW_RANGE } from '../server.constants';
+import { DC_2_ISO } from '../../servers.constants';
+
+export default class DedicatedServerDashboard {
+  /* @ngInject */
+  constructor(
+    $q,
+    $scope,
+    $rootScope,
+    $state,
+    $stateParams,
+    $translate,
+    atInternet,
+    Alerter,
+    constants,
+    DedicatedServerFeatureAvailability,
+    Server,
+    coreConfig,
+  ) {
+    this.$q = $q;
+    this.$scope = $scope;
+    this.$rootScope = $rootScope;
+    this.$state = $state;
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.atInternet = atInternet;
+    this.Alerter = Alerter;
+    this.constants = constants;
+    this.DedicatedServerFeatureAvailability = DedicatedServerFeatureAvailability;
+    this.Server = Server;
+    this.coreConfig = coreConfig;
+  }
+
+  $onInit() {
+    this.$rootScope.serverCtrl = this.dedicatedServer;
```

Comment:
I couldn't find any other solution.
Because without it, if you go from the dashboard to the progress page it works, but if you go from the dashboard page to the choice installation then template installation and finally progress installation you lose the service and it doesn't work.